src/main/java/de/retest/recheck/review/ignore/matcher/ElementXpathMatcher.java

```diff
@@ -0,0 +1,53 @@
+package de.retest.recheck.review.ignore.matcher;
+
+import java.util.regex.MatchResult;
+import java.util.regex.Pattern;
+
+import de.retest.recheck.review.ignore.io.RegexLoader;
+import de.retest.recheck.ui.descriptors.Element;
+
+public class ElementXpathMatcher implements Matcher<Element> {
```

Comment:
ElementX**P**athMatcher since Path is a [separate word](https://de.wikipedia.org/wiki/XPath).

Reply:
Tests?